addons/wiki/tests/test_wiki.py

```diff
@@ -38,14 +37,14 @@
 pytestmark = pytest.mark.django_db
 
 # forward slashes are not allowed, typically they would be replaced with spaces
-SPECIAL_CHARACTERS_ALL = u'`~!@#$%^*()-=_+ []{}\|/?.df,;:''"'
-SPECIAL_CHARACTERS_ALLOWED = u'`~!@#$%^*()-=_+ []{}\|?.df,;:''"'
+SPECIAL_CHARACTERS_ALL = r'`~!@#$%^*()-=_+ []{}\|/?.df,;:''"'
+SPECIAL_CHARACTERS_ALLOWED = r'`~!@#$%^*()-=_+ []{}\|?.df,;:''"'
```

Comment:
Making a note to double check this later.  Is the backslash supposed to be copied over?  I think so, but I'm not positive.

Reply:
Checked locally, seems to be good.